fractions/fluentd/src/main/java/org/wildfly/swarm/fluentd/FluentdFraction.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2015-2016 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.fluentd;
+
+import org.wildfly.swarm.config.logging.Level;
+import org.wildfly.swarm.spi.api.Fraction;
+
+/**
+ * @author Heiko Braun
+ */
+public class FluentdFraction implements Fraction<FluentdFraction> {
+
+    public FluentdFraction() {
+        this.hostname = "localhost";
```

Comment:
Do we need host/port?

I was under the assumption that FluentD collected logs by default, we didn't have to publish them somewhere?